File: Homework/Objective-C/HW8/HW8/Student.m
//
//  Student.m
//  HW8
//
//  Created by Руслан on 20.09.16.
//  Copyright © 2016 Ruslan Timchenko. All rights reserved.
//

#import "Student.h"

@implementation Student

- (instancetype)initWithName:(NSString*)name andSurname:(NSString*)surname
{
    self = [super init];
    if (self) {
        self.name = name;
        self.surname = surname;
    }
    return self;
}

- (NSString*) sayHello {
    return @"Hello!";
}

@end

File: Homework/Swift/HW8/HW8/Student.swift
//
//  Student.swift
//  HW8
//
//  Created by Руслан on 20.09.16.
//  Copyright © 2016 Ruslan Timchenko. All rights reserved.
//

import Foundation

class Student: NSObject {
    
    let name: String
    let surname: String
    
    init(name: String, surname: String) {
        self.name = name
        self.surname = surname
    }
    
    func sayHello() -> String {
        return "Hello!"
    }
}